libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.m

```diff
@@ -470,6 +366,26 @@ - (NSString*)uniqueUserAccountIdentifier {
     return identifier;
 }
 
+-(SFOAuthCredentials *)currentCredentials {
```

Comment:
@khawkins Good catch. Its not being used anywhere. We can do away with currentUserIdentity.